manageiq-graphql.gemspec

```diff
@@ -7,19 +7,19 @@ require "manageiq/graphql/version"
 Gem::Specification.new do |s|
   s.name        = "manageiq-graphql"
   s.version     = ManageIQ::GraphQL::VERSION
-  s.authors     = ["Chris Arcand"]
-  s.email       = ["chris@chrisarcand.com"]
-  s.homepage    = "https://github.com/manageiq/manageiq-graphql"
+  s.authors     = ["ManageIQ Authors"]
+  s.homepage    = "https://github.com/ManageIQ/manageiq-graphql"
   s.summary     = "The GraphQL API for ManageIQ"
-  s.description = "This project includes the Rails engine powering the GraphQL V2 API for ManageIQ - https://github.com/manageiq/manageiq"
+  s.description = "This project includes the Rails engine powering the GraphQL V2 API for ManageIQ - https://github.com/ManageIQ/manageiq"
   s.license     = "Apache-2.0"
 
   s.files = Dir["{app,config,db,lib}/**/*", "LICENSE.txt", "Rakefile", "README.md"]
 
   s.add_runtime_dependency "graphql", "~> 1.7"
   s.add_runtime_dependency "graphql-batch", "~> 0.3.8"
   s.add_runtime_dependency "graphql-preload", "~> 1.0"
-  s.add_runtime_dependency "rails", ">= 5.0", "< 5.1" # (In lieu of depending on ManageIQ core directly)
```

Comment:
Remove this because as-is, it causes security issues with Hakiri.  We can add back in, but I'd like to understand why it's needed.

Reply:
Just passing by...

If this is needed (probably to allow the tests to function... but guessing here), is there a version of `5.0.x` that we can use that satisfies Hakiri?  For reference, ManageIQ is pinned at `5.0.2`, so it seems like we could just change it to:

```ruby
s.add_runtime_dependency "rails", ">= 5.0.2", "< 5.1"
```

Reply:
Yes, or `s.add_runtime_dependency "rails", "~> 5.0.2", "< 5.1"`, I think

There is some weird bug with Hakiri where `"~> 5.0.2", "< 5.1"` works but `"< 5.1", "~> 5.0.2"` doesn't... but that's minor.

Reply:
This isn't required. I really did just add it in lieu of depending on ManageIQ core. I don't like that this gem (or any ManageIQ plugin) doesn't actually state every one of it's dependencies so I added this for fun. 

Since it actually has a negative effect as Jason stated, this is good 👌